### PR TITLE
README: reword mention of the API key being mandatory

### PR DESCRIPTION
**Pull request by chouquette**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,13 +48,13 @@ To deploy the Datadog Agent on hosts, add the Datadog role and your API key to y
     datadog_api_key: "<YOUR_DD_API_KEY>"
 ```
 
-The API key is required and its absence would cause the role to fail. If you want to provide it through another way, outside of Ansible's control, you'll still need to specify a dummy key and substitute it at a later point.
+The API key is required and its absence causes the role to fail. If you want to provide it through another way, outside of Ansible's control, specify a dummy key and substitute the key at a later point.
```
Comment (maycmlee) [inline README.md]:
```suggestion
The API key is required and its absence causes the role to fail. If you want to provide it through another way, outside of Ansible's control, specify a placeholder key and substitute the key at a later point.
```

**Review by maycmlee — COMMENTED** [review]
Just a small edit for inclusive wording.

File: README.md (hunk 1 above)
Comment (chouquette) [reply README.md]: I wasn't sure about it, now I'll know :)

File: README.md [hunk 2]
```diff
@@ -48,13 +48,13 @@ To deploy the Datadog Agent on hosts, add the Datadog role and your API key to y
...
 
 | Variable                                    | Description                                                                                                                                                                                                                                                                                                                                                                                                                                       |
 |---------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `datadog_api_key`                           | Your Datadog API key. This variable is mandatory.|
+| `datadog_api_key`                           | Your Datadog API key. **This variable is mandatory starting from 4.21**|
```
Comment (maycmlee) [inline README.md]:
```suggestion
| `datadog_api_key`                           | Your Datadog API key. **This variable is mandatory starting from 4.21**.|
```

**Review by maycmlee — APPROVED** [review]
Missed a small edit, but approving.

**Comment by chouquette** [comment]
Both suggestions applied, thanks!
And sorry again for merging the first one before your review :grimacing:

**Pull request merged**